src/components/AccountIcon.js

```diff
@@ -49,17 +50,32 @@ export default function AccountIcon(props) {
 			.catch(console.error);
 	};
 
+	if (address === '') {
```

Comment:
It's probably better to change the address proptype to be potentially undefined and do a null check. but that's just bike shedding.

Reply:
In places like `emptyAccount()` we have the default value of address to empty string. And In React if we render address in `<Text>`, an undefined error would appear. This is the main concern to use empty string like this situation as the default value for the string props.